core/src/definitions.ts

```diff
@@ -72,7 +72,37 @@ export interface PluginConfig {
   name: string;
 }
 
+export enum ExceptionCode {
+  /**
+   * API is not implemented.
+   *
+   * This means the API can't be used because:
+   *   - it is not implemented at all
```

Comment:
I don't think this will ever be the case (not implemented at all), when it's not implemented it's just because it's not possible in one or more platforms, but if it's not possible on any platform the method should just not exist.